CMakeLists.txt

```diff
@@ -226,11 +226,12 @@ SET(QT_USE_QTHELP        TRUE)
 ##
 
 if (BUILD_ALSA)
-      PKGCONFIG (alsa 1.0.0 ALSA_INCDIR ALSA_LIBDIR ALSA_LIB ALSA_CPP )
+      PKGCONFIG (alsa ALSA_INCDIR ALSA_LIBDIR ALSA_LIB ALSA_CPP )
```

Comment:
build output doesn't print the version #, but according to https://packages.debian.org/jessie/libasound2 it must be 1.0.28-1